about.html

```diff
@@ -0,0 +1,127 @@
+            <li><a href="#" onclick="hideMenu()">News</a></li>
+          </ul>
+          <a href="" class="camp" onclick="hideMenu()">GDC Campaign</a>
+        </div>
+      </nav>
+      <ul class="linage">
+        <li><a href="#"><img src="/assets/home.png" alt="home"></a></li>
+        <li><img src="/assets/arrow_left.png" alt="arrow_left"></li>
+        <li><a href="#">Events</a></li>
+        <li><img src="/assets/arrow_left.png" alt="arrow_left"></li>
+        <li><a href="#">GDC 2022</a></li>
+      </ul>
+      <div class="about-text">
+        <div class="texts">
+          <p class="hello">"Hello! Sharing world"</p>
+          <p class="title">GAME DEVELOPERS CONFERENCE SHOWCASE 2022</p>
+        </div>
+        <p class="sub-text">The Game Developers Conference (GDC) brings the game development community together to exchange ideas, solve problems, and shape the future of the industry across five days of education, inspiration, and networking. Attendees include programmers, artists, producers, game designers, audio professionals, and business leaders. GDC also hosts semi-annual installments of the GDC Masterclass program, which is comprised of day-long or multi-day, small-group virtual workshops that deliver in-depth, hands-on training around some of the most important challenges facing game developers today.</p>
+        <div class="contact-me">
+            <p>Please contact us per Email for any further questions about GDC 2022!</p>
+            <a href="mailto:gdcregistration@informa.com">gdcregistration@informa.com</a>
+        </div>
+      </div>
+    </header>
+    <!-- logo info section -->
+    <section class="logo-info">
+        <h2 class="logo-title">GDC 2022 Logo</h2>
+        <div class="logo-text">
+          <p>The Game Developers Conference began as an informal gathering of 25 developers in the living room of game designer Chris Crawford.</p>
+          <p class="hidden-text">In 2016, GDC commemorated the 30th edition of the conference. Year after year, GDC continues to define market innovations and provide insight into the future of games.</p>
+        </div>
+        <img src="/assets/frame-logo.png" alt="logo">
+    </section>
+    <!-- past conference -->
+    <section class="past">
+      <h2 class="logo-title">See the past Game Developers Conference</h2>
+      <p>Take a look at the last two conferences that took place in San Francisco, CA USA</p>
+      <div class="past-pic">
+        <div class="pic-col">
+          <img src="/assets/show1.jpg" alt="show1">
+          <div class="layer">
+            <h3>2019</h3>
+            <h4>GDC 2019 in San Francisco, CA USA</h4>
+          </div>
+        </div>
+        <div class="pic-col">
+          <img src="/assets/show2.jpg" alt="show1">
+          <div class="layer">
+            <h3>2020</h3>
+            <h4>GDC 2020 in San Francisco, CA USA</h4>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
Kindly make the following corrections for your `about` page :

- [ ] Please include the background images as seen in the design given as yours looks really plain and doesn't really have a good user interface 😄 
- [ ] Also, kindly take note of the font sizes and coors used in the paragraphs in the main body of the about section as yours seems quite small and not legible for user with an eye sight defficiency, remeber to alsways consider different type of users while building your site
- [ ] The margin betwen the contact section of the about page and the next section after it is quite alot, can you kindly reduce it ?
- [ ] As i said earlier, you seem to have an inconsistency in your font-type. I think you should use `lato` for headings and `cocogoose` for paragraphs to enable a legible design
- [ ] kindly increase the width of the logo you attached in the `LOGO SECTION` of your about page and remeber to give it a border box property too 😄 
- [ ] If you take a close look at the pictures attachedin the `See the past game developers conference` section of your project, it should have a background color property, with the date and venue positioned on top of it, please implement these changes to get a better look

Your design : 

https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/8ef5cc58-9081-4fe0-b7ca-20904988b40f


Behance design :

https://github.com/Fombi-Favour/microverse-capstone-1/assets/20537712/7b55fc79-7501-47a6-b7dc-91e3d00e5b15